packages/babel-plugin-proposal-object-rest-spread/src/index.js

```diff
@@ -131,6 +131,48 @@ export default declare((api, opts) => {
     });
   }
 
+  function createObjRefExpression(path, objRef) {
+    let impureDefaultValueDeclarator;
```

Comment:
nit: consider rename it to `nonStaticDefaultValueDeclarator` because it is defined when `t.isStatic(path)` is `false`.